include/libzfs.h

```diff
@@ -826,6 +827,8 @@ extern int zfs_mount(zfs_handle_t *, const char *, int);
 extern int zfs_mount_at(zfs_handle_t *, const char *, int, const char *);
 extern int zfs_unmount(zfs_handle_t *, const char *, int);
 extern int zfs_unmountall(zfs_handle_t *, int);
+/* True iff the system supports delegations or user is root */
```

Comment:
This comment is out of place, confusing, and incorrect.

Reply:
How so?

Reply:
Comments on functions in the header like this are not common.
The comment is accurate in the sense that you made the function do that, but I don't think it's what the function should do. FreeBSD may support delegation, but mounting isn't necessarily allowed for users.